vllm/compilation/backends.py

```diff
@@ -247,11 +247,6 @@ def __call__(self, graph: fx.GraphModule, example_inputs) -> Callable:
         self.split_gm, self.piecewise_graphs = split_graph(
             graph, self.compilation_configs.splitting_ops)
 
-        from torch._dynamo.utils import lazy_format_graph_code
-        logger.debug("%s", lazy_format_graph_code("before split", self.graph))
-        logger.debug("%s", lazy_format_graph_code("after split",
-                                                  self.split_gm))
```

Comment:
Going to revert this, but I deleted these because they made the logs very noisy and difficult to read while I was trying to debug an unrelated error. Maybe we can add a `debug_verbose` logging level and use it here?

Reply:
I'd be in favor of deleting these.  I think we should switch these over to use the same `dump_graph` system used for compilation passes.

Reply:
Ok, I'm going to leave them deleted then :)

Reply:
please leave it unchanged, i will improve it at https://github.com/vllm-project/vllm/pull/10972 .